apps/ledger-live-mobile/src/newArch/features/LandingPages/screens/LargeMoverLandingPage/utils/index.tsx

```diff
@@ -50,4 +52,19 @@ const getColors = (priceChangePercentage: number) => {
     ? { textColor: "success.c70", bgColor: "success.c10" }
     : { textColor: "error.c50", bgColor: "error.c10" };
 };
-export { formatCounterValue, getTimeAgoCode, getColors };
+
+const rangeMap: Record<string, KeysPriceChange> = {
+  hour: KeysPriceChange.hour,
+  day: KeysPriceChange.day,
+  week: KeysPriceChange.week,
+  month: KeysPriceChange.month,
+  year: KeysPriceChange.year,
+};
+
+function getCurrencyIdsFromTickers(tickers: string[]): string[] {
+  return tickers
+    .map(findCryptoCurrencyByTicker)
+    .filter(Boolean)
+    .map(currency => (currency ? currency.id : ""));
+}
```

Comment:
```suggestion
function getCurrencyIdsFromTickers(tickers: string[]): string[] {
  return tickers.flatMap(ticker => {
    const currency = findCryptoCurrencyByTicker(ticker);
    return currency?.id ? [currency.id] : [];
  });
}
```

Using `.flatMap()` here simplifies the logic by combining filtering and transformation into a single step,  
while avoiding empty strings when `findCryptoCurrencyByTicker` returns `undefined`.